oaknational/Oak-Web-Application
# Feat/lesq 395/remember homepage tab [LESQ-395]

## Description

- update url with hash when selecting tabs on the homepage in order to store the state for browser navigation
- let me know if there is a more idiomatic nextjs way to do this! 

## How to test

1. Go to https://deploy-preview-2031--oak-web-application.netlify.thenational.academy
2. Click on curriculum tab
3. follow a link to another page (eg early release units)
4. click back on the browser
5. You should see the curriculum tab is active

## ACs

- [x]  The app ‘remembers’ which homepage tab the user came from when the user returns to the homepage via the back button.
- [x]  No extra browser history entries are added as a result of this change.
